client/src/components/login/RegisterForm.vue

```diff
@@ -6,33 +6,57 @@
-                        <b-card-body>
-                            <b-form-group label="Email Address">
-                                <b-form-input v-model="email" name="email" type="text" />
-                            </b-form-group>
-                            <b-form-group label="Password">
-                                <b-form-input v-model="password" name="password" type="password" />
-                            </b-form-group>
-                            <b-form-group label="Confirm password">
-                                <b-form-input v-model="confirm" name="confirm" type="password" />
-                            </b-form-group>
-                            <b-form-group label="Public name">
-                                <b-form-input v-model="username" name="username" type="text" />
-                                <b-form-text
-                                    >Your public name is an identifier that will be used to generate addresses for
-                                    information you share publicly. Public names must be at least three characters in
-                                    length and contain only lower-case letters, numbers, dots, underscores, and dashes
-                                    ('.', '_', '-').</b-form-text
-                                >
-                            </b-form-group>
-                            <b-form-group
-                                v-if="mailing_join_addr && server_mail_configured"
-                                label="Subscribe to mailing list">
-                                <input v-model="subscribe" name="subscribe" type="checkbox" />
-                            </b-form-group>
-                            <b-button name="create" type="submit" :disabled="disableCreate">Create</b-button>
-                        </b-card-body>
+                    <b-card no-body>
+                        <!-- OIDC/External Login enabled: encourage users to use it instead of local registration -->
+                        <span v-if="custosEnabled">
+                            <b-card-header v-b-toggle.accordion-oidc role="button">
+                                Register using institutional account
+                            </b-card-header>
+                            <b-collapse visible id="accordion-oidc" role="tabpanel" accordion="registration_acc">
+                                <b-card-body>
+                                    Create a Galaxy account using an institutional account (e.g.:Google/JHU). This will
+                                    redirect you to your institutional login through Custos.
+                                    <external-login :login_page="false" />
+                                </b-card-body>
+                            </b-collapse>
+                        </span>
+                        <!-- Local Galaxy Registration -->
+                        <b-card-header v-b-toggle.accordion-register role="button">
```

Comment:
This allows collapse even when only this is shown -- it should be disabled in that case.
![image](https://user-images.githubusercontent.com/155398/166584080-499b6a18-4ada-49e8-a74d-5f1054362102.png)

Reply:
Should be fixed in this next commit, registration is no longer a `b-collapse` when Custos is not preferred (and not enabled).